src/metrics-logger.ts

```diff
@@ -0,0 +1,77 @@
+'use strict';
+import { Request } from './custom-typings';
+import { Route, RouteSpec, Response, Server } from 'restify';
+import { StatsD, StatsDOptions } from './statsd';
+
+/**
+ * tuple of [seconds, nanoseconds]
+ */
+type HighResolutionTime = [number, number];
+
+export const DEFAULT_KEY_NAME: string = 'handler-0';
+
+export function register(server: Server, statsDOptions: StatsDOptions) {
```

Comment:
looking at the implementation of this in the teammember-api and feature-flag-api, it is not clear at all what is being registered. can we rename to something like `registerHandledRouteTimingMetrics`?